docs/wiki/frameworks/custom-signal-func.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Custom radio signal strengh functions
+---
+
+ACRE2 allows replacing its default signal strength calculator with a custom signal strength function. This can be done by using the following API function `acre_api_fnc_setCustomSignalFunc`. This page addresses aims to assist in creating your own and provide an examples.
+
+The default signal calculator is sophisticated and complex and as such is largely calculated in the C++ extension. However this signal model doesn't always appeal to every community and it doesn't always afford possibilites for various mission mechanics such as radio jammers. This command is intended to be used by communities seeking a specific signal expierence or for highly ambitious mission makers.
+
+For the time being this command is highly expiermental and is included as of dev-build 940. 
+
+## Function Usage
+
+You can see the default function here but note this mainly just a dispatcher that sends data to the C++ extension. It gives an idea as to what data might be useful. It does not provide insight into the default signal model that is used though.
+
+### Input
+
+The input to your custom function will be of the format: `[30, 5000, "ACRE_PRC343_ID_1", "ACRE_PRC343_ID_2"]`
+- The first element is the frequency (megahertz) e.g. `30`. 
+- The second element is the power of the transmitter (milliwatts) e.g. `5000`,.
+- Then the classname of the recieving radio e.g. `"ACRE_PRC343_ID_1"`.
+- Lastly the classname of the recieving radio e.g. `"ACRE_PRC343_ID_2"`.
+
+### Output
+
+The output of your function should be an array of size 2 with two numerical values:
+- The signal strength as a percentage (0-1). This value is what will be used by the teamspeak plugin to adjust the audio of the player being heard on the radio.
+- The decibal signal strength value (dBM). A typical value that is heard is between 0 to ~ -110 (radio specific). Lower values are not heard.
+
+The decibal signal strength value value is used to determine if the specific radio is capabable of hearing the transmission. This process is typically carried out in the `handleMultipleTransmissions` function for that particular radio radio (i.e. AN/PRC-148). Firstly there exists a minimum signal sensitivity that each radio can pick up - This can be thought of as the minimum signal strength the hardware is capabable of registering. The minimum that is registered is typically around -100 (AN/PRC343) to -117 (AN/PRC-117F) this is defined in `configFile >> 'CfgAcreComponents' >> RADIO_BASECLASS >> 'sensitivityMin'`. Some radios also make use of Squelch - If you are curious to see this it is all handled in the radio specific handleMultipleTransmissions function.
```

Comment:
`decibel`
`capable`
`which is defined in`
`to see this is all handled`
missing code ticks around last `handleMultipleTransmissions`